autorest/codegen/models/operation.py

```diff
@@ -307,6 +305,19 @@ def convert_multiple_content_type_parameters(self) -> None:
         chosen_parameter.multiple_content_types_docstring_type = docstring_type
         self.parameters.append(chosen_parameter)
 
+    @property
+    def use_json_object(self) -> bool:
+        return self.code_model.options["version_tolerant"] and (
+                self.parameters.has_body or
```

Comment:
Instead of defining this on the operation, and having a utils function `import_mutable_mapping` that you use in multiple places, if you define the mypy type `JSON` in the `def imports` of `HiddenModelObjectSchema`, and include all of the other needed definitions and imports there as well.

Now, the operations will get it automatically, because if you look at `def imports` for operations, we get the imports from parameters and responses as well, which get imports from their schema. So if we say "HiddenModelObjectSchema" needs JSON defined and all of the imports needed for that, then it bubbles up to the top of the page

Reply:
Thanks for detail advice, I will update tomorrow.

Reply:
Hi @iscai-msft , I updated to make `HiddenModelObjectSchema.imports` the unique place to import mutableMappings. It does help avoid redundant codes.